x/paychan/model.go

```diff
@@ -39,19 +44,28 @@ func (pc *PaymentChannel) Validate() error {
 
 // Copy returns a shallow copy of this PaymentChannel.
 func (pc PaymentChannel) Copy() orm.CloneableData {
-	return &pc
+	return &PaymentChannel{
+		Metadata:     pc.Metadata.Copy(),
+		Src:          pc.Src.Clone(),
+		SenderPubkey: pc.SenderPubkey,
+		Recipient:    pc.Recipient.Clone(),
+		Total:        pc.Total.Clone(),
+		Timeout:      pc.Timeout,
+		Memo:         pc.Memo,
+		Transferred:  pc.Transferred.Clone(),
+	}
 }
 
 // PaymentChannelBucket is a wrapper over orm.Bucket that ensures that only
 // PaymentChannel entities can be persisted.
 type PaymentChannelBucket struct {
-	orm.Bucket
+	migration.Bucket
 	idSeq orm.Sequence
 }
 
 // NewPaymentChannelBucket returns a bucket for storing PaymentChannel state.
 func NewPaymentChannelBucket() PaymentChannelBucket {
-	b := orm.NewBucket("paychan", orm.NewSimpleObj(nil, &PaymentChannel{}))
+	b := migration.NewBucket("paychan", "paychan", orm.NewSimpleObj(nil, &PaymentChannel{}))
```

Comment:
hmmm... same name twice? do we want to always use the same id for both cases?

Reply:
It is a paychan package and the model is called paychan. Naming is hard :see_no_evil: 

I think the package name should stay paychan as it is the package name. Although it will be a bit of work, I can rename the model. Any suggestions for the new name?